spot-ml/src/main/scala/org/apache/spot/SuspiciousConnectsArgumentParser.scala

```diff
@@ -95,6 +96,10 @@ object SuspiciousConnectsArgumentParser {
       action((x, c) => c.copy(hdfsScoredConnect = x)).
       text("HDFS path for results")
 
+    opt[String]("tempmodel").required().valueName("<hdfs path>").
```

Comment:
Could we take advantage of @natedogs911 's python substitute of ml_ops.sh and create a settings.txt file on the fly? That's just from the top of my head. In general I see what you are saying.